### PR TITLE
feat/US34248-US34253: DAO Enhancements

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**
**Description**:
- Added "DAODashboard" common component to MultiSig.
- Added Member / Admin indicator to DAO's
- Fixed DAO Description Field for Gov and NFT DAO.
- Added social links to DAO's Overview Page.
- Added social links to DAO's Setting Page.

Screenshot 2023-06-19 at 4 05 41 PM

### CODE REVIEW

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx [hunk 1]
```diff
@@ -12,7 +12,8 @@ export function GovernanceDAODashboard() {
 
   const accountTokenBalancesQueryResults = useAccountTokenBalances("");
   const { data: tokenBalances } = accountTokenBalancesQueryResults;
-
+  const { data: daoGovTokenBalance = 0 } = useTokenBalance({ tokenId: dao?.tokenId ?? "" });
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx:15]: In GOV DAO an admin can have 0 or more than zero tokens in that case what will be the condition for the admin.?
- Is it necessary for the Admin to have GOV DAO Token if yes should it be more than zero to show user as admin.?

**Review by roshan-bisht-lab49 — COMMENTED** [review]
Added one doubt regarding the the `admin Indicator` for Gov DAO.

File: src/dex-ui/pages/dao/utils.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export function getDaoLinksArray(linksString: string): string[] {
```
Comment (roshan-bisht-lab49) [inline src/dex-ui/pages/dao/utils.ts:1]: In Future SC team is removing the key support for DAO links, we would be getting the array direct from SC rather a key value string.

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx (hunk 1 above)
Comment (damiano-melcarne-lab49) [reply src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx:15]: I'm not 100% sure - please forward this question to Raj. My guess is that "Admin" tag should show up if the connected wallet matches the `adminId` in the solidity events regardless if they have gov tokens in their wallet. I would also ask Raj and the SC team the exact list of expected and existing privileges an admin has for the Governance and NFT DAOs. That may inform how we handle the UX for the Member/Admin tag.

File: src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx [hunk 3]
```diff
@@ -1,56 +1,88 @@
...
-            gap="8"
-          >
-            <Text textStyle="p medium semibold">Assets</Text>
-            <Flex direction="row" justifyContent="space-between">
-              <MetricLabel label="TOTAL ASSETS" value={totalAssetDisplay} />
-              <MetricLabel label="TOKENS" value={tokenCountDisplay} />
+        <Grid templateColumns="repeat(2, 1fr)" gap={2}>
+          <GridItem>
+            <Flex
+              flex="1"
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx:23]: Use `layerStyles` when there are repeated style patterns (`dao-dashboard__card`). These boxes should have the same styles across all DAO dashboard types.

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx (hunk 1 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboard.tsx:17]: Sure, for now I have kept it like if you admin no matter if you have GOV token or not Admin Indicator will come up.

File: src/dex-ui/pages/dao/DAODashboard.tsx [hunk 4]
```diff
@@ -47,17 +22,74 @@ interface DAODashboardProps extends PropsWithChildren {
...
+            title: "Members",
+          },
+          {
+            icon: <SettingsIcon boxSize="5" />,
+            title: "Settings",
+          },
+        ];
+      case DAOType.GovernanceToken:
+      case DAOType.NFT:
+        return [
```
[the remark this answers is not in the document]
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAODashboard.tsx]: Done

File: src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx (hunk 3 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/MultiSigDAODashboard/DashboardOverview.tsx]: Done

File: src/dex-ui/pages/dao/DAODashboard.tsx [hunk 5]
```diff
@@ -47,17 +22,54 @@ interface DAODashboardProps extends PropsWithChildren {
...
   const tabIndexByRoute = daoNavigationTabs.map((tab) => tab.title.toLowerCase()).indexOf(currentTabNameByRoute);
-  const intialTabIndex = tabIndexByRoute === -1 ? 0 : tabIndexByRoute;
-  const { handleTabChange } = useTabFilters(intialTabIndex);
+  const initialTabIndex = tabIndexByRoute === -1 ? 0 : tabIndexByRoute;
+  const { handleTabChange } = useTabFilters(initialTabIndex);
 
+  function GetDAONavigationTabs() {
+    switch (type) {
+      case DAOType.MultiSig:
+        return daoTabs.filter((tab) => tab.title !== "Assets");
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/DAODashboard.tsx:65]: Should be `tab.title !== "Staking"`

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]

File: src/dex-ui/pages/dao/DAODashboard.tsx (hunk 5 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAODashboard.tsx]: Oops my bad, fixed it.

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Pull request merged**